dotcom-rendering/src/components/AppsEpic.importable.tsx

```diff
@@ -0,0 +1,125 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import type { IGetEpics__Result } from '@guardian/bridget/Acquisitions';
+import {
+	body as bodySizes,
+	from,
+	headline,
+	palette,
+	remSpace,
+	textSans,
+} from '@guardian/source-foundations';
+import { useEffect, useState } from 'react';
+import { getAcquisitionsClient, getUserClient } from '../lib/bridgetApi';
+import { EpicContent } from './EpicContent.apps';
+
+export const AppsEpic = () => {
+	const [isPremium, setIsPremium] = useState<boolean | undefined>(undefined);
+	const [maybeEpic, setMaybeEpic] =
+		useState<IGetEpics__Result['success']>(undefined);
+
+	useEffect(() => {
+		void getUserClient()
+			.isPremium()
+			.then(setIsPremium)
+			.catch(() => undefined);
+	}, []);
+
+	useEffect(() => {
+		void getAcquisitionsClient()
+			.getEpics()
+			.then((me) => {
+				setMaybeEpic(me);
+			})
+			.catch(() => undefined);
```

Comment:
As above I think this .catch is unreachable because of the `void` keyword. We could move the catch inside the .then if we wanted to catch any errors whilst setting `me` into state